prismic-model/diffCustomTypes.ts

```diff
@@ -29,12 +30,34 @@ export default async function diffContentTypes(
     })
     .filter(d => d);
 
-  if (deltas.length > 0) {
-    error(`Diffs found on ${deltas.map(delta => delta.id).join(', ')}`);
+  const sliceDeltas = sharedSlices
+    .filter(({ local }) => typeof local !== 'undefined')
+    .map(({ id, local, remote }) => {
+      const delta = diffString(
+        remote,
+        removeUndefinedProps(local), // we'll never get undefined props from Prismic, so we don't want them locally
+        {
+          keepUnchangedValues: true,
+        }
+      );
+
+      if (delta.length > 0) {
+        printDelta(id, delta);
+        return { id };
+      }
+      return null;
+    })
+    .filter(d => d);
```

Comment:
More of "by the way" than anything, when I applied the same typescript rules across, that one became an error (haven't figure out which rule for yet), and [this](https://github.com/wellcomecollection/wellcomecollection.org/pull/10968/files#diff-5b457cb21d093ba33ebc04f033df1c75266f00da73d55a37d2045b98cc8e48a9) was the fix